STEERING-COMMITTEE.md

```diff
@@ -7,8 +7,8 @@ aliases:
   - /contributing/steering-committee/
 ---
 
-The Knative Steering Committee (KSC) is the ultimate authority for the Knative
-project, and governs all aspects of the project.
+The Knative Steering Committee (KSC) is
+[responsible for the general health of the Knative community](./GOVERNANCE.md).
 
 The governance of Knative is an open, living document, and will continue to
 evolve as the community and project change. We expect over time we will adapt
```

Comment:
I suggest we delete the sentence 'We expect over time we will adapt...' from both here and the steering document, the sentence about it being an open living document being sufficient; and that we add the 'open, living' sentence to the TOC so that all three are symmetrical.

Reply:
I'm not sure that the TOC charter needs the bit about governance, because the TOC is supposed to be focusing on the technical outcomes, not the governance processes.

I can take out the "we expect" as long as we keep in "will continue to evolve" -- I want to make sure that the documents make clear that they are not a final state so much as a step on the journey.